lib/modules/datasource/postprocess-release.ts

```diff
@@ -0,0 +1,38 @@
+import { logger } from '../../logger';
+import { getDatasourceFor } from './common';
+import type { PostprocessReleaseConfig, Release } from './types';
+
+export async function postprocessRelease(
+  config: PostprocessReleaseConfig,
+  release: Release,
+): Promise<Release | null> {
+  const { datasource } = config;
+
+  if (!datasource) {
+    return release;
+  }
+
+  const ds = getDatasourceFor(datasource);
+  if (!ds) {
+    return release;
+  }
+
+  if (!ds.interceptRelease) {
+    return release;
+  }
+
+  try {
+    const result = await ds.interceptRelease(
+      {
+        packageName: config.packageName,
+        registryUrl: config.registryUrl,
+        currentValue: config.currentValue,
```

Comment:
Are we sure currentValue is necessary/relevant?

Reply:
Yep, it only causes confusion at this pount, removing it